shared/chat/inbox/row/small-team/container.tsx

```diff
@@ -64,7 +64,8 @@ export default Container.namedConnect(
     const participantNeedToRekey = stateProps._meta.rekeyers.size > 0
     const youNeedToRekey = !participantNeedToRekey && stateProps._meta.rekeyers.has(stateProps._username)
     const isDecryptingSnippet =
-      (hasUnread || stateProps.snippet.length === 0) && Constants.isDecryptingSnippet(stateProps._meta)
+      (hasUnread || (stateProps.snippet?.length ?? 0) === 0) &&
```

Comment:
this shouldn't be the fix. the typing of this is that snippet is always a string so the type plumbing is wahts actually broken